src/mountscript.sh

```diff
@@ -196,9 +200,43 @@ search_free_local_ip_with_prefix()
     fi
     
     local local_ip=""
+    local optimize_get_free_local_ip=false
+    IFS=$'\n' 
+    local used_local_ips_with_same_prefix=$(cat $MOUNTMAP | awk '{print $2}' | grep "^$initial_ip_prefix" | sort -t . -k 1,1n -k 2,2n -k 3,3n -k 4,4n)
 
     _3rdoctet=100
     ip_prefix=$initial_ip_prefix
+
+    #
+    # Optimize the process to get free local IP by starting the loop to choose
+    # 3rd and 4th octet from the number which was used last and still exist in
+    # MOUNTMAP instead of starting it from 100.
+    #
+    if [ ! -z "$used_local_ips_with_same_prefix" -a $OPTIMIZE_GET_FREE_LOCAL_IP = true ]; then
+        
+        last_used_ip=$(echo "${used_local_ips_with_same_prefix##*$'\n'}")
```

Comment:
change this to the more readable.
last_used_ip=$(echo "$used_local_ips_with_same_prefix" | tail -n1)

also, 
-n "$used_local_ips_with_same_prefix" in place of '! -z "$used_local_ips_with_same_prefix"'

there are few more suggestions, pls use the following lime

if [ "$OPTIMIZE_GET_FREE_LOCAL_IP" == "true" -a -n "$used_local_ips_with_same_prefix" ]